### PR TITLE
User data and computation state clearing

### LINKED ISSUE #59: State clearing via the UI
Add a button in the UI that clears local state. It should have some sort of warning indicating to users that the action deletes all projects.

### PR DESCRIPTION
**Pull request by swashcap**
- **Problem:** Clearing persisted data (on disk) requires running npm scripts. It isn't easy!
- **Solution:** Provide a settings page and buttons to clear the data.
- **Testing:**
  1. Boot _coinstac-ui_ in development mode
  2. Log in
  3. Observe stored data in `$HOME/.coinstac/<username>/`
  4. Go to the settings page
  5. Click the “Delete user data” button
  6. Observe stored data is removed

Closes #59.

### CODE REVIEW

**Comment by swashcap** [comment]
UI updates per e5d362ac4d96b0ba1695338bfad3ad95c5ea6041:

delete-em

**Comment by swashcap** [comment]
Computation deletion dropped in 83fd22391b16382c797f26d705f544a68473e648:

drop-comps

File: packages/coinstac-ui/app/main/services/clean.js [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+        return rimrafAsync(dir).then(() => app.core.teardown());
+      }
+    });
+}
+
+module.exports = {
+  computations: function cleanComputations() {
+    return warnAndDelete(
+      {
+        detail: 'Computations can be re-downloaded',
```
Comment (rssk) [inline packages/coinstac-ui/app/main/services/clean.js]: Does this make sense now we've bundled things?

File: packages/coinstac-ui/app/main/services/clean.js [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+        message: 'Delete saved computations',
+      },
+      app.core.getComputationsDirectory()
+    );
+  },
+
+  userData: function cleanUserData(username) {
+    return warnAndDelete(
+      {
+        detail: 'You can not undo this action',
```
Comment (rssk) [inline packages/coinstac-ui/app/main/services/clean.js:28]: I think we should mention this deletes projects, as its probably not clear what user data is to the enduser

File: packages/coinstac-ui/app/main/services/clean.js (hunk 1 above)
Comment (swashcap) [reply packages/coinstac-ui/app/main/services/clean.js]: Good point. I'll take it out.

File: packages/coinstac-ui/app/main/services/clean.js (hunk 2 above)
Comment (swashcap) [reply packages/coinstac-ui/app/main/services/clean.js:28]: Cool, I’ll update this. I think clarification belongs in the Settings UI, not the alert.

File: packages/coinstac-ui/app/main/services/clean.js (hunk 1 above)
Comment (swashcap) [reply packages/coinstac-ui/app/main/services/clean.js]: Dropped in a7625dbd64ae28ef7ac8e3b086c56acbd906b146.

File: packages/coinstac-ui/app/main/services/clean.js (hunk 2 above)
Comment (swashcap) [reply packages/coinstac-ui/app/main/services/clean.js:28]: Updated in 34764acb4cbab9b1c9254f99d2963c9c64c2cb8d.

**Pull request merged**